SoundSwitch/Util/TrayIcon.cs

```diff
@@ -102,6 +103,19 @@ public void Dispose()
             _updateMenuItem.Dispose();
         }
 
+        private void InitIcon()
+        {
```

Comment:
Can you add a check for
*AppConfigs.Configuration.KeepSystrayIcon*
in the initIcon method. If true, just return directly from the method.